services/webos-service-remote/subscription.js

```diff
@@ -0,0 +1,41 @@
+import EventEmitter from 'events';
+import { Message } from './message';
+
+//* Subscription is an EventEmitter wrapper for subscribed LS2 calls
+export class Subscription extends EventEmitter {
+    constructor(handle, uri, args) {
+        super();
+        this.uri = uri;
+        this.args = args;
+        this.handle = handle;
+        this.request = handle.subscribe(uri, JSON.stringify(args));
```

Comment:
I think you may be doing double JSON-stringify - both here and in `handle.subscribe`? Or is the second one intentionally used as a  boneless shell escape?

Regardless, I would rather have the formatting handled as close to final string joining as possible. (ie. directly in `handle.subscribe`) Partially due to this confusion - you could just add a sensible comment that'd explain what's actually happening if that's the expected behaviour :D

Reply:
Yes! You are right, and this has been ixed.